service Modeling {
    rpc GetUpdate (UpdateRequest) returns (UpdateResponse) {}
    rpc GetMap () returns (Map) {}
}

message Metadata {}

message UpdateRequest {
    Metadata meta = 1;
}

message UpdateResponse {
    Metadata meta = 1;
}

message Map {
    Metadata meta = 1;
}